### DISCUSSION TITLE
Maintain API key across multiple user accounts in Windows

### DISCUSSION THREAD

**Discussion by trikydiky** [timmo001/system-bridge#504] [Help]
Hi,

Currently trying to configure System Bridge to trigger automations in homeassistant.

One limitation I've come across is that the API key changes when switching between users in Windows requiring reconfiguration in homeassistant.

Is there a way to maintain the API key across different user accounts?
I tried to clone the install directory in AppData\Local\system-bridge between the users as well as running System Bridge from the a different users AppData\ directory but the API key still changed.

Any advise would be much appreciated.

Many thanks

**Comment by majorsl** [comment]
I just installed this and have the same question.  Not being able to have it running across multiple user accounts in Windows is a real dealbreaker with Home Assistant.

**Comment by nicotjo1** [comment]
Got the same issue
